privacy-security-questionnaire.md

```diff
@@ -0,0 +1,97 @@
+# [Self-Review Questionnaire: Security and Privacy](https://w3ctag.github.io/security-questionnaire/)
+
+> 01.  What information does this feature expose,
+>      and for what purposes?
+
+None. This specification is for a header that an origin outputs to the user agent, for optimizing page loading performance.
+
+ * What information does your spec expose to the first party that the first party cannot currently easily determine: None.
+ * What information does your spec expose to third parties that third parties cannot currently easily determine: None
+ * What potentially identifying information does your spec expose to the first party that the first party can already access (i.e., what identifying information does your spec duplicate or mirror): None.
+ * What potentially identifying information does your spec expose to third parties that third parties can already access: None.
+
+> 02.  Do features in your specification expose the minimum amount of information
+>      necessary to implement the intended functionality?
+
+Yes. The feature only exposes the minimum required information, as a boolean header, to satisfy the use case.
+
+> 03.  Do the features in your specification expose personal information,
+>      personally-identifiable information (PII), or information derived from
+>      either?
+
+No. The feature does not relate to PII or any derived information thereof.
+
+> 04.  How do the features in your specification deal with sensitive information?
+
+No. The feature does not deal with sensitive information.
+
+> 05.  Does data exposed by your specification carry related but distinct
+>      information that may not be obvious to users?
+
+Yes. The specification carries related information about the page, specifically if there are linked resources to be expected in the requested page, as a Document-Policy header in the response header, which will not be obvious to users.
```

Comment:
Reading https://w3ctag.github.io/security-questionnaire/#hidden-data in full, I think "This feature does not allow users to share data with origins" is probably a better answer.